src/Inventory/Asset/Unmanaged.php

```diff
@@ -310,17 +286,6 @@ public function handleLinks(array $data = null)
         return parent::handleLinks();
     }
 
-    public function getManagementPorts()
```

Comment:
public methods should not be removed in bugfixes versions.

```php
/**
 * @FIXME Remove method in GLPI 10.1.
 */
public function getManagementPorts()
{
    return [];
}
```

Reply:
done